docs/simulation/calculations.yaml

```diff
@@ -36,9 +32,34 @@ POPs:
     population_growth: Sum of population_growth modifiers
     pop_growth: Sum of pop_growth modifiers
     global_population_growth/10: Sum of global_population_growth modifiers
-  Consciousness from literacy:
-    formula: defines.pops.CON_LITERACY * Plurality * POP literacy * (1 + literacy_con_impact)
-  Life needs scalar:
-    formula: (1 + Plurality) * (1 + 2 * Consciousness / defines.pops.PDEF_BASE_CON) * (1 + sum of goods_demand) * defines.pop.BASE_GOODS_DEMAND * POP size / 200000
-  Everyday & luxury needs scalar:
-    formula: Life needs scalar * (1 + number of inventions * defines.pops.INVENTION_IMPACT_ON_DEMAND)
\ No newline at end of file
+  Attitudes:
+    Consciousness from literacy: defines.pops.CON_LITERACY * Plurality * POP literacy * (1 + literacy_con_impact)
+  Needs:
+    Life needs scalar: (1 + Plurality) * (1 + 2 * Consciousness / defines.pops.PDEF_BASE_CON) * (1 + sum of goods_demand) * defines.pop.BASE_GOODS_DEMAND * POP size / 200000
+    Everyday & luxury needs scalar: Life needs scalar * (1 + number of inventions * defines.pops.INVENTION_IMPACT_ON_DEMAND)
+  POP income:
+    Pensions: 2 * pension_level * administrative efficiency * social spending slider * base life needs price * (1 - effective tax) * POP size / 200000
+    Unemployment subsidies: 2 * unemployment_benefit * administrative efficiency * social spending slider * base life needs price * (1 - effective tax) * POP size / 200000
+    Wages:
+      Minimum (worker) wage: 2 * minimum_wage * administrative efficiency * base life needs price * (1 - effective tax) * POP size / 200000
+      RGO wages:
+        Owners:
+          if minimum worker wages > normal worker wage:
+            formula: (RGO income - total worker income) * (1 - effective tax)
```

Comment:
Shouldn't this include a `owner POP size / owners` term, like for the `else` case below and for both worker cases, so that `RGO income - total worker income` is divided between owner POPs proportional to their size? Otherwise it'll create money out of thin air based on the number of distinct owner POPs, e.g. if a single POP of 1000 owners gets paid `£ x`, then the same population split into two POPs of size 500 would get paid a total of `£ 2x`.

Reply:
You're right. I forgot to add that.
Background: I copied some old notes and edited them slightly. I missed this line.